docs/conf/portland/2023/health.rst

```diff
@@ -0,0 +1,72 @@
+:template: {{year}}/generic.html
+
+Health and Safety Policy
+========================
+
+This policy is up to date as of **November 2022**.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+**Vaccine or Negative COVID-19 Test Requirement**
+-------------------------------------------------
+
+All attendees **must show proof of COVID-19 vaccination OR a negative COVID-19 test** upon first entry to the event. Attendee badges will be used as proof of vaccination or negative test validation throughout the event, each participant will need to pick up their own badge.
+
+**Vaccine Requirement**
+
+If you are showing proof of vaccination to attend in person, you will need to show that you are fully vaccinated against the COVID-19 virus. We require that you:
+
+* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+* that your most recent dose was at least fourteen days before the conference start
+* that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+* Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
+* Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
+**Negative COVID-19 Test** **Requirement**
+
+In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
```

Comment:
What is a verified provider? Is there a list?

Reply:
I think basically just from a healthcase provider, but we can be more explicit here.